src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/service/FlowPathBuilder.java

```diff
@@ -153,6 +151,8 @@ public FlowPath buildFlowPath(Flow flow, PathResources pathResources, Path path,
                 .bandwidth(flow.getBandwidth())
                 .ignoreBandwidth(flow.isIgnoreBandwidth() || forceToIgnoreBandwidth)
                 .latency(path.getLatency())
+                .srcWithMultiTable(switchProperties.get(srcSwitch.getSwitchId()).isMultiTable())
```

Comment:
According to code that fills switchProperties - it is possible to have no switch properties object by some existing switchId... and as result, you will throw NPE here.

Reply:
Fixed.